locale/ar/site.json

```diff
@@ -153,5 +160,18 @@
     "pages": {
       "changelog": "سجل التغييرات"
     }
+  },
+  "banners": {
+    "index": {
+      "startDate": "2021-08-25T16:00:00.000Z",
```

Comment:
If you leave out the date, it will use the one from the English. That might mean the text gets out of sync when the English changes, but one less thing to update